src/pages/index.js

```diff
@@ -108,9 +109,9 @@ const IndexPage = ({ data, pageContext }) => {
               <Button
                 as={ExternalLink}
                 variant={Button.VARIANT.PRIMARY}
-                href="https://newrelic.com/signup?partner=Developer+Edition"
+                href="https://newrelic.com/signup/?utm_source=developer-site"
```

Comment:
It looks like the `/signup/` page (with the trailing `/`) doesn't work. Can you remove the trailing `/`?

Reply:
Ah, copied and pasted from the doc. Kinda surprised that doesn't work. Either way, updating it now (via suggestion)
```suggestion
                href="https://newrelic.com/signup?utm_source=developer-site"
```

Reply:
Ya not sure why it doesn't work. I happened to see something mentioned in one of the slack channels.